src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,54 @@
+import { debugLogger } from '../utils/logging.js';
+import { AnalyticsData, AnalyticsHandler } from './sdk/analyticsHandler.sdk.js';
+import { getTelemetrySessionId, saveTelemetrySessionId } from './session.js';
+import { v4 as uuidv4 } from 'uuid';
+
+export class GenezioTelemetry {
+
+  static async getSessionId(): Promise<string> {
+    const sessionId = await getTelemetrySessionId();
+    if (!sessionId) {
+      const newSessionId = uuidv4();
+      debugLogger.debug(`[GenezioTelemetry]`, `New session id: ${newSessionId}`);
+      saveTelemetrySessionId(newSessionId);
+      return newSessionId;
+    }
+    
+    return sessionId;
+  }
+
+  public static async sendEvent(eventType: string, cl?: string, errorTrace?: string): Promise<void> {
```

Comment:
```suggestion
  public static async sendEvent(eventType: string, cloudProvider?: string, errorTrace?: string): Promise<void> {
```

I find using short name versions hard to read. Is this intentional? If not, I think we should use more suggestible naming. This comment also applies to `ul`, `opS` and other more to come.

Reply:
i think you are right. i will change to full names